packages/core/utils/src/event-bus/index.ts

```diff
@@ -0,0 +1,47 @@
+const listeners: Listener[] = [];
+let emitted: Event[] = [];
+
+type Handler = (value?: any) => void;
```

Comment:
as this is exposed as external param, should be exported as well
```suggestion
export type Handler = (value?: unknown) => void;
```

Reply:
noo this is only for internal usage, it's not used anywhere